File: frontendapp/lib/models/cinema.dart
class Cinema {
  final int id;
  final String name;
  final String city;
  final String coordinates;
  final String address;
  final String phoneNumber;
  final int maxRoom;
  String? imageName;

  Cinema({
    required this.id,
    required this.name,
    required this.city,
    required this.coordinates,
    required this.address,
    required this.phoneNumber,
    required this.maxRoom,
    this.imageName,
  });

  factory Cinema.fromJson(Map<String, dynamic> json) => Cinema(
    id: json['id'],
    name: json['name'],
    city: json['city'],
    coordinates: json['coordinates'],
    address: json['address'],
    phoneNumber: json['phonenumber'],
    maxRoom: json['maxroom'],
    imageName: json['imagename'],
  );
}

File: backend/bookingmovieticketapp/src/main/java/com/project/bookingmovieticketapp/Models/Cinema.java
package com.project.bookingmovieticketapp.Models;

import jakarta.persistence.*;
import lombok.*;

@Entity
@Setter
@Getter
@AllArgsConstructor
@NoArgsConstructor
@Table(name = "cinemas")
@Builder
public class Cinema {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int id;

    @Column(name = "name", nullable = false)
    private String name;

    @Column(name = "city",nullable = false)
    private String city;

    private String coordinates;

    @Column(name = "address", nullable = false)
    private String address;

    private String phonenumber;
    private int maxroom;
}
